src/rust/Cargo.toml

```
@@ -24,6 +24,7 @@ bytes = { version = "1.5.0", optional = true }
 log = "0.4.20"
 fallible-streaming-iterator = "0.1.9"
 tempfile = "3.8.1"
+reqwest = { version = "0.11.22", optional = true }
```

It might be preferable to relax the pin here to `0.11`? 0.11.22 is latest. Or maybe that doesn't matter. http-range-client is set to 0.11:
https://github.com/pka/http-range-client/blob/19d25265b9172fdbdd1d2b5dfd095e90c22c2408/Cargo.toml#L24
```suggestion
reqwest = { version = "0.11", optional = true }
```

---

It's preferable to use the oldest version you have actually tested and verified works. In practice this is usually the latest version when developing. There's marginal utility in supporting older versions. 

Note that in rust,version pinning is different from some other versioning schemes and will use the latest semver compatible with this notation. 

https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277/1